tests/xrt/02_simple/main.cpp

```diff
@@ -20,33 +20,22 @@
 #include <string>
 #include <cstring>
 #include <sys/mman.h>
-#include <time.h>
-#include <chrono>
-#include <thread>
 
-// driver includes
+// XRT includes
+#include "xrt.h"
 #include "ert.h"
-
-// host_src includes
-#include "xclhal2.h"
+#include "experimental/xrt_kernel.h"
+#include "experimental/xrt_xclbin.h"
```

Comment:
At this point, XRT tests with the experimental APIs won't work on the sprite because libxrt++ is not official library (for now) and sprite does not include libxrt++ for host compilation. So, either we need to disable this test on the sprite or find a way to add libxrt++ for this test.

Reply:
These APIs are defined in xrt_coreutil.so which is available so the tests should work. The library is currently included in common.mk makefile

Reply:
The sprite is not using common.mk for compilation. It is using metadata to generate the makefile for the sprite run. It's because the existing makefile finds the common.mk from upper directory, which can't be done in sprite. So, it looks like I'll have to do necessary changes for the sprite.